magento/magento2
# Introduced MediaGalleryRenditions and MediaGalleryRenditionsApi modules

### Description (*)

Introduced MediaGalleryRenditions and MediaGalleryRenditionsApi modules

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/321
https://github.com/magento/partners-magento2-infrastructure/pull/34


### Fixed Issues (if relevant)

Fixes https://github.com/magento/adobe-stock-integration/issues/1286

### Approval Request

https://jira.corp.magento.com/browse/MC-37383

### Manual testing scenarios (*)
1. Insert image from media gallery to content - rendition instead of original image should be insterted
2. `Stores -> Configuration -> System -> Media Gallery Renditions` configuration change should update all existing renditions (`bin/magento queue:consumers:start media.gallery.renditions.update`) and used for new renditions creation
3. Rendition should be removed from filesystem when the corresponding asset is removed

### Contribution checklist (*)
